bin/improver-probabilities-to-realizations

```diff
@@ -96,6 +96,11 @@ def main():
              'random numbers used for splitting tied values '
              'within the raw ensemble, so that the values from the input '
              'percentiles can be ordered to match the raw ensemble.')
+    reordering.add_argument(
+        '--ecc_bounds_warning', default=False, action='store_true',
+        help='Option to specify that exceeding ECC bounds will '
+        'only result in a warning message rather than an '
+        'exception.')
```

Comment:
As above (perhaps noting that percentiles are the intermediate output before realizations).

Reply:
comment changed